src/ops/stash.rs

```diff
@@ -33,25 +34,31 @@ impl OpTrait for StashWorktree {
     fn get_action(&self, _target: Option<&TargetData>) -> Option<Action> {
         let update_fn = move |state: &mut State, term: &mut Term| -> Res<()> {
             if state.prompt.state.status().is_done() {
+                if is_working_tree_empty(&state.repo)? {
```

Comment:
There's no need to initiate a prompt when `is_working_tree_empty`.
See `state.prompt.set` below at line 70

Reply:
Yeah, that makes sense. Do we want to show something to user? Like "Cannot stash: working tree is empty"

Reply:
Sounds like a good idea!

Reply:
I have encountered very strange situation.., can you try to run current tests and after that test things locally? Locally I am getting expected error message when working tree is empty, but there is no message in test. Did I miss something?

Reply:
The test tries to `zw` and then input `test` and Enter.
It breaks after the change with the prompt I noticed. Pressing keys while the error is up will close it :) Line 500

Reply:
My bad 🤦‍♂️ Thanks for taking a look!